### ISSUE TITLE
Breaking a tree from it's roots doesn't play a falling animation, and also don't yields any drops

### ISSUE THREAD

**Issue by Brun333rp** [ferreusveritas/Dynamic-Trees#576]
**Describe the bug**
When choping down a tree through it's roots, it will disappear automatically without any traces, instead of playing a falling down animation. It also won't yield any drops most of the time, only some occasional accorns but nothing more than this afaik.

**To Reproduce**
Chop down a any through it's roots.

**Rendering Glitches**
"Are you using OptiFine (or any equivalent mods that mess with rendering)?"
Answer: Yes. But I don't believe this is the cause of the problem.

**Comment by Harleyoc1** [comment]
This is intended behaviour. Break it from the branch, not the roots.

**Comment by Brun333rp** [comment]
I don't see how it's intended, but since that's in this way...

**Comment by Brun333rp** [comment]
maybe there are technical reasons for that? because there's no way that it would be designed in this way, it doesn't makes any sense. you may say that this is just a game and that not everything needs to make sense, but isn't the point of the mod making trees realistic? because a tree simply vanishing by breaking it through it's roots is not realistic at all

**Comment by Brun333rp** [comment]
I've discovered that it affects all trees, so I've changed the title.

**Comment by democat3457** [comment]
> maybe there are technical reasons for that? because there's no way that it would be designed in this way, it doesn't makes any sense. you may say that this is just a game and that not everything needs to make sense, but isn't the point of the mod making trees realistic? because a tree simply vanishing by breaking it through it's roots is not realistic at all

Well you aren’t even supposed to be able to break the tree by it’s root block, so it just acts like a tree deletion - plus, have you ever heard of chopping down a tree by its roots instead of the trunk/branches in real life?

**Comment by Brun333rp** [comment]
guess I've understand the reason why it's intended, as you are not supposed to break the tree through the roots. also, previously, this issue was just about swamp oaks, but then I discovered that it affects all the trees, so I've changed the title and description so other people can search about it here and see that it's intended.
 
about the root-breaking behaviour, the roots of a regular tree are under the trunk and most times are not exposed, so players would instinctively chop down the tree through the trunk instead. but the roots of the swamp oak are exposed, so many players would likely attemp to chop down the tree through them thinking that it's the base of the trunk, wich is the why I've made this issue. I've kinda suspected that it was intended though, but then I decided to make a issue about this to make sure
